From 6432f05d57236a3279f92d51082e7eb1db3a6953 Mon Sep 17 00:00:00 2001
From: Aster Boese <asterboese@mailbox.org>
Date: Tue, 26 Dec 2023 20:45:54 -0500
Subject: [PATCH] manually disable tests

Fixes: 
Cannot find source file:

/usr/src/gtest/src/gtest-all.cc

---
 CMakeLists.txt | 15 ---------------
 1 file changed, 15 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 7a08d6f..c5d2fbe 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -48,21 +48,6 @@ include_directories (SYSTEM ${SERVICE_DEPS_INCLUDE_DIRS})
 set (CMAKE_INCLUDE_CURRENT_DIR ON)
 set (CC_WARNING_ARGS " -Wall -Wextra -Wno-missing-field-initializers")
 
-# testing & coverage
-if (${enable_tests})
-  set (GTEST_SOURCE_DIR /usr/src/gtest/src)
-  set (GTEST_INCLUDE_DIR ${GTEST_SOURCE_DIR}/..)
-  set (GTEST_LIBS -lpthread)
-  enable_testing ()
-  if (${enable_lcov})
-    include(GCov)
-  endif ()
-endif ()
-
 add_subdirectory (src)
 add_subdirectory (data)
 add_subdirectory (po)
-
-if (${enable_tests})
-  add_subdirectory (tests)
-endif ()
-- 
2.43.0

